@types/window.d.ts

```diff
@@ -1,5 +1,6 @@
 import type { TeamSubscription } from '../libs/@guardian/libs/src/logger/@types/logger';
 import type { Switches } from '../libs/@guardian/libs/src/switches/@types/Switches';
+import { ImaManager } from 'libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer';
```

Comment:
this path is wrong, which makes me thing the `declare` below was still making it ambient (assuming nothing errored?)

```suggestion
import { ImaManager } from '../libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer';
```